benchmarks/benchmark_registration.py

```diff
@@ -50,7 +51,7 @@ def setup(self, ndims, image_size, upsample_factor, dtype, *args):
         if phase_cross_correlation is None:
             raise NotImplementedError("phase_cross_correlation unavailable")
         shifts = (-2.3, 1.7, 5.4, -3.2)[:ndims]
-        phantom = img_as_float(data.binary_blobs(length=image_size, n_dim=ndims))
+        phantom = rescale_to_float(data.binary_blobs(length=image_size, n_dim=ndims))
```

Comment:
With the update, this now always scales float arrays to the target range [0, 1]. Still need to check if this affects any of these gallery examples in a negative way.